console/src/components/CompactHeader/CompactHeader.less

```diff
@@ -0,0 +1,13 @@
+.compactHeader {
+    // font-family: SourceSansPro;
+    .title {
+        font-size: 20px;
+        letter-spacing: 12.5px;
```

Comment:
I think this will be a theme-specific rule. If using `agate/Divider` makes sense as I suggested above, I think we should remove this here and try to decide how best to support theme-specific rules like this via our existing skinning system.